.github/workflows/ios.yml

```diff
@@ -0,0 +1,23 @@
+name: iOS
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build and Test FlipperZero(ios) scheme using iPhone 11 simulator
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Xcode build
+        run: |
+          sudo xcode-select --switch /Applications/Xcode_12.app
+          xcodebuild -sdk iphoneos -workspace ./FlipperZero/FlipperZero.xcodeproj/project.xcworkspace -scheme 'FlipperZero(iOS)'  clean archive -archivePath ./output/build/archive/FlipperZeroiOS.xcarchive CODE_SIGNING_ALLOWED=NO
+      - name: Test
+        run: |
+          xcodebuild -sdk iphonesimulator -workspace ./FlipperZero/FlipperZero.xcodeproj/project.xcworkspace -scheme 'FlipperZero(iOS)' test DSTROOT=./output/test/build.dst OBJROOT=./output/test/build.obj SYMROOT=./output/test/build.sym SHARED_PRECOMPS_DIR=./output/test/build.pch -destination platform='iOS Simulator,name=iPhone 11'
```

Comment:
I guess output of `xcodebuild` might be too verbose. Could you please add `xcpretty`?

Reply:
Haven't used that before. That made the output way better